README.md

```diff
@@ -125,15 +126,16 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
 ## Contributing
 
-Do you think you have found a bug? Then please report it via the GitHub issue tracker. Make sure to
-attach any problematic files that can be used to reproduce the issue. Such files are also used to
-create regression tests that ensure that your bug will never return.
+Do you think you have found a bug? Then please report it via the GitHub issue
+tracker. Make sure to attach any problematic files that can be used to reproduce
+the issue. Such files are also used to create regression tests that ensure that
+your bug will never return.
 
-When submitting pull requests, please prefix your commit messages with `fix:` or `feat:` for bug
-fixes and new features respectively. This is the
-[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) scheme that is used to
-automate some maintenance chores such as generating the changelog and inferring the next version
-number.
+When submitting pull requests, please prefix your commit messages with `fix:` or
+`feat:` for bug fixes and new features respectively. This is the
+[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) scheme
+that is used to automate some maintenance chores such as generating the
+changelog and inferring the next version number.
```

Comment:
Just adding the UFID frame to the feature list is enough, please revert the formatting applied to the rest of the file :)

Reply:
Sorry, autoformatters in my editor got ahead of me.